dymos/utils/misc.py

```diff
@@ -253,3 +255,26 @@ def create_subprob(base_name, comm, reports=False):
         str_hash = hashlib.sha256(used_for_security=False)[:8]
         p = om.Problem(comm=comm, reports=reports, name=f'{sim_prob_name}_{str_hash}')
     return p
+
+
+def om_version():
```

Comment:
Removes a lot of duplicate code of converting OpenMDAO version to tuple for comparison.